platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/container/ContainerNode.java

```
@@ -0,0 +1,286 @@
+            throwIfIn(LifeCycle.DESTROYED, "Node has already been destroyed.");
+
+            log.info("Starting node {}...", selfId);
+
+            final StartRequest startRequest = StartRequest.newBuilder()
+                    .setSelfId(ProtobufConverter.toGoogle(selfId))
+                    .setRoster(ProtobufConverter.toGoogle(roster))
+                    .setKeysAndCerts(KeysAndCertsConverter.toProto(keysAndCerts))
+                    .putAllOverriddenProperties(nodeConfiguration.overriddenProperties())
+                    .build();
+
+            final TestControlStub stub = TestControlGrpc.newStub(channel);
+            stub.start(startRequest, new StreamObserver<>() {
+                @Override
+                public void onNext(final EventMessage value) {
+                    if (value.hasPlatformStatusChange()) {
+                        final PlatformStatusChange change = value.getPlatformStatusChange();
+                        final String statusName = change.getNewStatus();
+                        try {
+                            final PlatformStatus newStatus = PlatformStatus.valueOf(statusName);
+                            platformStatus = newStatus;
+                            resultsCollector.addPlatformStatus(newStatus);
+                        } catch (final IllegalArgumentException e) {
+                            log.warn("Received unknown platform status: {}", statusName);
+                        }
+                    } else if (value.hasLogEntry()) {
+                        receivedLogs.add(ProtobufConverter.fromGoogle(value.getLogEntry()));
+                    } else if (value.hasConsensusRounds()) {
+                        resultsCollector.addConsensusRounds(ProtobufConverter.fromGoogle(value.getConsensusRounds()));
+                    } else {
+                        // Other message types can be handled later
+                        log.info(value);
```

We should throw an error here IMO. I believe this can only happen if the app and `ContainerNode` are out of sync.